ext/gmp/gmp.c

```diff
@@ -164,10 +164,11 @@ static zend_result convert_zstr_to_gmp(mpz_t gmp_number, const zend_string *val,
 static zend_result convert_to_gmp(mpz_t gmpnumber, zval *val, zend_long base, uint32_t arg_pos);
 static void gmp_cmp(zval *return_value, zval *a_arg, zval *b_arg, bool is_operator);
 
-static bool gmp_zend_parse_arg_into_mpz(
+static bool gmp_zend_parse_arg_into_mpz_ex(
 	zval *arg,
 	mpz_ptr *destination_mpz_ptr,
-	uint32_t arg_num
+	uint32_t arg_num,
+	bool is_operator
```

Comment:
Why not pass 0 directly? Seems simpler

Reply:
Because I would be reusing the same zpp_arg storage if I use 0

Reply:
oh right